apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -713,6 +680,49 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
             });
     };
 
+    /**
+     * Render the totp QR code regenerate confirmation modal
+     */
+    const renderConfirmRegenerateModal = (): ReactElement => {
+        return (
+            <Modal
+                data-testid={ `${testId}-regenerate-confirm-modal` }
+                size="small"
+                open={ isConfirmRegenerationModalOpen }
+                closeOnDimmerClick={ false }
+                dimmer="blurring"
+            >
+                <Modal.Content data-testid={ `${testId}-regenerate-confirm-modal-content` }>
```

Comment:
refrain from using `data-testid` use `data-componentid` instead.

Reply:
check other places

Reply:
This is a component we had earlier, not a new one. So it has already used testids and I think if we change them, a lot of e2e might break.

Reply:
better to add a `data-componentid` param then, keeping the `data-testid`, and create an issue to check the E2E on this